main.go

```diff
@@ -199,10 +199,11 @@ func GetDirectoryContext(workingDir string, paths []string, isFile bool) []pathV
 	if len(paths) == 0 {
 		// create appName directory if downloading whole app
 		addPathVals := pathVal{
-			RootWorkingDirectoryLocal: filepath.FromSlash(localPath + appName + "/"),
+			RootWorkingDirectoryLocal: localPath + appName + "/",
 			StartingPathServer:        startingPath,
 		}
 
+		addPathVals.RootWorkingDirectoryLocal = filepath.FromSlash(addPathVals.RootWorkingDirectoryLocal)
```

Comment:
You **really** shouldn't have to do this. I understand that you tried and encountered problems the other way, but it baffles me. What gave you the hint to try doing this in the first place?